src/content/en/2020/markup.md

```diff
@@ -373,19 +400,19 @@ In our mobile data set of 6.3 million pages, around 0.9 million pages (14.01%) c
 <figcaption>{{ figure_link(caption="Obsolete elements with more than 10,000 uses.", sheets_gid="1972617631", sql_file="pages_element_count_by_device_and_obsolete_elements.sql") }}</figcaption>
 </figure>
 
-Why are these still alive—even `spacer` is still being used 1,584 times, and present on every 5,000th page. We know that Google has been using a `center` element on [their homepage](https://www.google.com/) [for 22 years](https://web.archive.org/web/19981202230410/https://www.google.com/) now, but does that find so many imitators?
+Why are these still alive? Even `spacer` is still being used 1,584 times, and present on every 5,000th page. We know that Google has been using a `center` element on [their homepage](https://www.google.com/) [for 22 years](https://web.archive.org/web/19981202230410/https://www.google.com/) now, but why are there so many imitators?
+
+#### `isindex`
 
-<div class="note">
-  <h4><code>isindex</code></h4>
-  <p>If you were wondering: The <a href="https://www.w3.org/TR/html401/interact/forms.html#edef-ISINDEX"><code>isindex</code> element</a> was present once. (It was part of the HTML specs <a href="https://meiert.com/en/indices/html-elements/">until version 4.01, and of XHTML in 1.0</a>, yet only properly <a href="https://lists.w3.org/Archives/Public/public-whatwg-archive/2006Feb/0111.html">specified in 2006</a>, and <a href="https://github.com/whatwg/html/pull/1095">removed in 2016</a>.)</p>
-</div>
+{# TODO(authors): What is the significance of this section? There are no new stats provided. #}
+If you were wondering: The [`isindex`](https://www.w3.org/TR/html401/interact/forms.html#edef-ISINDEX) element was present once. It was part of the HTML specs until [version 4.01](https://meiert.com/en/indices/html-elements/), and of XHTML in 1.0, yet only properly [specified](https://lists.w3.org/Archives/Public/public-whatwg-archive/2006Feb/0111.html) in 2006, and [removed](https://github.com/whatwg/html/pull/1095) in 2016.
 
 ### Proprietary and made-up elements
 
-In our set of elements we found some that were neither standard HTML (nor SVG nor MathML) elements, nor custom ones, nor obsolete ones, but somewhat proprietary ones. The Top 10 that we identified were the following:
+In our set of elements we found some that were neither standard HTML (nor SVG nor MathML) elements, nor custom ones, nor obsolete ones, but somewhat proprietary ones. The top 10 that we identified were the following:
```

Comment:
1. “were” → “are” (twice)
2. “top” → “Top“ (preference—I’m used to this being capitalized, and I find it clearer)